### ISSUE TITLE
Enable Travis CI

### ISSUE THREAD

**Issue by DanielRuf** [olefredrik/FoundationPress#1321]
It seems Travis CI is not used and was last running 8 months ago.

https://travis-ci.org/olefredrik/FoundationPress/builds/354429059

Is there a reason to not use it? How could we use it and which tests should we run?

**Comment by derweili** [comment]
I enabled Travis CI and added PHP compatibility checks.

However, additional tests should be added:
- [ ] wpcs
- [ ] SASS
- [ ] JavaScript
- [ ] PHP Unit Tests?

**Comment by derweili** [comment]
I'm not sure what tests we should run for sass and JavaScript.
It should at least be tested whether everything can be compiled correctly.

Do further tests make sense?

**Comment by DanielRuf** [comment]
> I'm not sure what tests we should run for sass and JavaScript.
> It should at least be tested whether everything can be compiled correctly.

SassLint and ESLint probably (or more).

**Comment by kLOsk** [comment]
I agree on the linting, however we should make sure that they are not too strict. I work a lot with Sage and the new Sage9 version has ridiculously strong rules for the linting in place which are just annoying (especially in terms of whitespace and new lines). So in order to also be accessible to less professional developers (or people who like to copy and paste code), we should be a bit less restrictive. What do you think?
